pkg/api/index.go

```diff
@@ -28,6 +28,19 @@ var dataSourcesConfigurationAccessEvaluator = ac.EvalAll(
 	),
 )
 
+// dataSourcesNewAccessEvaluator is used to protect the "Configure > Data sources > New" page access
```

Comment:
@alexanderzobnin, @kalleep, @sakjur:
Not sure where I should actually put these evaluators. In the `roles.go` file, maybe? In their own `evaluator.go` file?
Wdyt?

Reply:
Do you use them anywhere else? I currently have a set of licensing read permissions that are being evaluated in both `licensing/api/api.go` and `api/frontendsettings.go`, so I could benefit from having an evaluator for them somewhere in OSS, but I'm not sure whether there's a good place for it.